presto-main/src/main/java/com/facebook/presto/split/BufferingSplitSource.java

```diff
@@ -45,6 +45,12 @@ public ConnectorId getConnectorId()
     }
 
     @Override
+    public Split getEmptySplit()
```

Comment:
Does this need to be in SplitSource? My feeling is that there's a better way, it makes the API kinda weird.